events/wc_society_events.txt

```diff
@@ -173,3 +173,267 @@ character_event = {
+		text = EVTDESC_WCCSO.10202_stewardship
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whispers_intrigue }
+		text = EVTDESC_WCCSO.10202_intrigue
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whispers_learning }
+		text = EVTDESC_WCCSO.10202_learning
+	}
+	picture = GFX_evt_madness
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whispers_trigger = yes }
+
+	option = {
+		name = EVTOPTA_WCCSO.10202
+		character_event = { id = WCCSO.10203 days = 90 random = 60 }
+		if = {
+			limit = { NOT = { trait = lunatic } }
+			random = {
+				chance = 66
+				add_trait = lunatic
+			}
+		}
+	}
+}
+
+character_event = {
+	id = WCCSO.10203
+	desc = EVTDESC_WCCSO.10203
+	picture = GFX_evt_madness
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whispers_trigger = yes }
+
+	option = {
+		name = EVTOPTA_WCCSO.10203
+
+    	if = {
+			limit = { has_character_flag = writing_whispers_martial }
+			if = {
+				limit = { has_character_flag = whispers_quality_1 }
+				add_artifact = whispers_martial_1
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_2 }
+				add_artifact = whispers_martial_2
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_3 }
+				add_artifact = whispers_martial_3
+			}
+			else = {
+				add_artifact = whispers_martial_4
+			}
+		}
+		else_if = {
+			limit = { has_character_flag = writing_whispers_stewardship }
+			if = {
+				limit = { has_character_flag = whispers_quality_1 }
+				add_artifact = whispers_stewardship_1
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_2 }
+				add_artifact = whispers_stewardship_2
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_3 }
+				add_artifact = whispers_stewardship_3
+			}
+			else = {
+				add_artifact = whispers_stewardship_4
+			}
+		}
+		else_if = {
+			limit = { has_character_flag = writing_whispers_intrigue }
+			if = {
+				limit = { has_character_flag = whispers_quality_1 }
+				add_artifact = whispers_intrigue_1
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_2 }
+				add_artifact = whispers_intrigue_2
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_3 }
+				add_artifact = whispers_intrigue_3
+			}
+			else = {
+				add_artifact = whispers_intrigue_4
+			}
+		}
+		else = {
+			if = {
+				limit = { has_character_flag = whispers_quality_1 }
+				add_artifact = whispers_learning_1
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_2 }
+				add_artifact = whispers_learning_2
+			}
+			else_if = {
+				limit = { has_character_flag = whispers_quality_3 }
+				add_artifact = whispers_learning_3
+			}
+			else = {
+				add_artifact = whispers_learning_4
+			}
+		}
+		remove_character_modifier = writing_whispers
+		clr_character_flag = wrote_magnum_opus
```

Comment:
You can write more than one book as one character. Is it intended?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75095562-fa455000-55a6-11ea-8f6e-6e13175c4c21.png)

</details>

Reply:
yes

Reply:
You can get artifacts with pretty big bonuses through life of just one character, but ok

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75095718-ebf83380-55a8-11ea-8b30-3bbbc5349d56.png)

</details>